projects/telemetry/datavisualization/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+LABEL author="Jack Greenberg"
```

Comment:
Lol I do not think my name should be here :laughing:

Reply:
fixed!